src/Integration.Vsix/SonarLintIntegrationPackage.cs

```diff
@@ -91,7 +90,6 @@ private async System.Threading.Tasks.Task InitOnUIThreadAsync()
                 var workspace = await this.GetMefServiceAsync<VisualStudioWorkspace>();
 
                 var vsSolution = serviceProvider.GetService<SVsSolution, IVsSolution>();
-                this.sonarAnalyzerManager = new SonarAnalyzerManager(activeSolutionBoundTracker, workspace, vsSolution, logger, sonarQubeIssuesProvider);
```

Comment:
This is a quick change to disable the old Roslyn suppression behaviour and make it easier to debug/test the new version.

Completely removing the old version is more complicated: there's quite a lot of old code to remove, and changes to some of the project files (we should be able to get rid out our references to the SonarAnalyzer assembly). Also, I need to make sure I don't break CFamily suppressions.